packages/examples/packages/manage-state/src/utils.ts

```diff
@@ -18,16 +18,23 @@ const DEFAULT_STATE = {
  *
  * This uses the `snap_manageState` JSON-RPC method to get the state.
  *
+ * @param encrypted - An optional flag to indicate whether to use encrypted storage or not.
  * @returns The current state of the snap.
  * @see https://docs.metamask.io/snaps/reference/rpc-api/#snap_managestate
  */
-export async function getState(): Promise<State> {
+export async function getState(encrypted?: boolean): Promise<State> {
   const state = await snap.request({
     method: 'snap_manageState',
 
-    // For this particular example, we use the `ManageStateOperation.GetState`
-    // enum value, but you can also use the string value `'get'` instead.
-    params: { operation: ManageStateOperation.GetState },
+    params: {
+      // For this particular example, we use the `ManageStateOperation.GetState`
+      // enum value, but you can also use the string value `'get'` instead.
+      operation: ManageStateOperation.GetState,
+      // By default all state is encrypted, but you can choose to not encrypt it
+      // To do this you may set this flag to false
+      // This will use a separate unencrypted storage from the encrypted state
```

Comment:
```suggestion
      // By default all state is encrypted, but you can choose to not encrypt it.
      // To do this you may set this flag to false.
      // This will use a separate unencrypted storage from the encrypted state.
```